Makefile

```diff
@@ -2,22 +2,25 @@
 SHELL := /bin/bash
 
 # versions at  https://github.com/kubernetes-sigs/controller-tools/releases
-CONTROLLER_GEN_VERSION = v0.10.0
+CONTROLLER_GEN_VERSION = v0.12.0
 
 # GO_VERSION refers to the version of Golang to be downloaded when running dockerized version
-GO_VERSION = 1.19
+GO_VERSION = 1.20
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.23
```

Comment:
Bump to 1.25 as we already use K8s v1.25 in [go.mod](https://github.com/medik8s/self-node-remediation/blob/main/go.mod#L18-L21)